openslides_backend/presenter/search_deleted_models.py

```diff
@@ -0,0 +1,89 @@
+from typing import Any, List
+
+import fastjsonschema
+from datastore.shared.util import DeletedModelsBehaviour
+
+from ..permissions.management_levels import OrganizationManagementLevel
+from ..permissions.permission_helper import has_organization_management_level
+from ..shared.exceptions import PermissionDenied
+from ..shared.filters import And, Filter, FilterOperator, Or
+from ..shared.schema import schema_version
+from .base import BasePresenter
+from .presenter import register_presenter
+
+schema = fastjsonschema.compile(
+    {
+        "$schema": schema_version,
+        "type": "object",
+        "properties": {
+            "collection": {"type": "string"},
+            "filter_string": {"type": "string"},
+            "meeting_id": {"type": "integer", "minimum": 1},
+        },
+        "required": ["collection", "filter_string"],
```

Comment:
The code made me assume you want the meeting_id required!?
Please, write the presenter documentation in the wiki.